CHANGELOG.md

```diff
@@ -34,10 +53,6 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Fixed a bug in integer validation of arrays that threw an error when an array contained a mix of strings and integers.
 
-- Fixed a bug in JupyterLab >= 4 and Jupyter Notebook >= 7 that caused latex to not render in plotly charts. 
```

Comment:
Are these corrections to the `5.24.1` changelog?

Reply:
They were added in error to the changelog file here https://github.com/plotly/plotly.py/pull/4763/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed after the release and don't appear in the release notes on the release tab

Reply:
Resolved.